runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewDelegate.java

```diff
@@ -35,9 +38,14 @@
     private static boolean sLibraryLoaded = false;
     private static boolean sRunningOnIA = true;
     private static final String PRIVATE_DATA_DIRECTORY_SUFFIX = "xwalkcore";
+    // In previous versions, MANDATORY_PAKS is actually not mandatory, it's just a list of
```

Comment:
I agree too. But since this file have been in this list for a long time, I think it's better to keep it for the moment and give some comments so that someone else could understand this change.